src/util-device.c

```diff
@@ -39,6 +39,23 @@ static int live_devices_stats = 1;
 static int LiveSafeDeviceName(const char *devname,
                               char *newdevname, size_t destlen);
 
+static int g_live_devices_disable_offloading = 1;
```

Comment:
I'm really ok with the global idea as running ethtool is painful. But if we disable offloading by default we have an impact on the system and returning to initial state is currently not done. So the question is, do we have user that could run from time to time suricata and suffer from performance loss due to the removal of offloading. I don't see much cases but we may have an important one which is firewall with optional IDS capability.